tests/topotests/all_protocol_startup/r1/ipv4_routes.ref

```diff
@@ -8,6 +8,16 @@ C>* 192.168.6.0/26 is directly connected, r1-eth6, XX:XX:XX
 C>* 192.168.7.0/26 is directly connected, r1-eth7, XX:XX:XX
 C>* 192.168.8.0/26 is directly connected, r1-eth8, XX:XX:XX
 C>* 192.168.9.0/26 is directly connected, r1-eth9, XX:XX:XX
+L>* 192.168.0.1/32 is directly connected, r1-eth0, XX:XX:XX
```

Comment:
NAK, `L` routes should always be on a loopback device (either `lo` or the VRF device)

Reply:
I don't understand this feedback. For one, loopback routes wouldn't need a `Local` representation since the kernel would either hit `table local` first (`lo`) or it would route the packet into the VRF's table where it would hit the corresponding `local` route - i.e. no special handling needed to make pings work when leaked. Secondly, L/Local routes for non-loopback interface IPs is how this is handled in other implementations (e.g. Cisco) - and these are what need to be leaked to address the ping issue we discussed in the meeting on Tuesday.

Reply:
Was talking with David privately prior to this comment and he has realized that this is not how it necessarily works in Linux.  I may end up putting the Local route on the loopback or not.  Not sure yet

Reply:
(further discussion on slack)